mne/io/brainvision/brainvision.py

```diff
@@ -33,9 +33,10 @@ class RawBrainVision(BaseRaw):
     ----------
     vhdr_fname : str
         Path to the EEG header file.
-    montage : str | None | instance of Montage
-        Path or instance of montage containing electrode positions.
-        If None, sensor locations are (0,0,0). See the documentation of
+    montage : str | True | None | instance of Montage
+        Path or instance of montage containing electrode positions. If
+        True, read embedded sensor locations. If None, sensor locations
+        are (0,0,0). See the documentation of
         :func:`mne.channels.read_montage` for more information.
```

Comment:
In that case, I suppose it makes sense that this information is read when `montage=None` (though I guess this information will typically exist because a `.bvef` file was loaded into the recording software in the first place). However, if I implement the above it will break the brainvision test at line 297 since now it will read electrode position info by default thus requiring the user to explicitly set channels without coordinates (or rather with `0,0,0`) to either `eog` or `misc`. This might not be a problem, I guess, because it will force the user to set the correct channel type. I can modify the test which fails, setting `misc=['ReRef']` (only the `eog` field is set in this set).